### PR TITLE
Added section with helpful Chatterino-related links to the About page

### LINKED ISSUE #2965: General wiki link
**Feature request**

Not sure if there's already an existing issue for this, but can there be a link to the Wiki somewhere? Maybe around the About button? 🤔
image

### PR DESCRIPTION
**Pull request by xHeaveny**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

Added section with helpful Chatterino-related links to the About page.
Closes #2965

### CODE REVIEW

File: src/widgets/settingspages/AboutPage.cpp [hunk 1]
```diff
@@ -128,7 +130,21 @@ AboutPage::AboutPage()
...
+        // About Chatterino
+        auto about_chatterino = layout.emplace<QGroupBox>("About Chatterino...");
+        {
+            auto l = about_chatterino.emplace<QVBoxLayout>();
 
+            // clang-format off
+            l.emplace<QLabel>("Chatterino Wiki can be found <a href=\"https://wiki.chatterino.com\">here</a>")->setOpenExternalLinks(true);
+            l.emplace<QLabel>("Support <a href=\"https://streamelements.com/fourtf/tip\">Chatterino</a>")->setOpenExternalLinks(true);
+            l.emplace<QLabel>("All about Chatterino's <a href=\"https://chatterino.com/#features\">features</a>")->setOpenExternalLinks(true);
+            l.emplace<QLabel>("Join the official Chatterino <a href=\"https:/\discord.gg/7Y5AYhAK4z\">Discord</a>")->setOpenExternalLinks(true);
```
Comment (pajlada) [inline src/widgets/settingspages/AboutPage.cpp:143]: The discord link is broken
```suggestion
            l.emplace<QLabel>("Join the official Chatterino <a href=\"https://discord.gg/7Y5AYhAK4z\">Discord</a>")->setOpenExternalLinks(true);
```

File: src/widgets/settingspages/AboutPage.cpp [hunk 2]
```diff
@@ -128,7 +130,21 @@ AboutPage::AboutPage()
                        "https://github.com/lamerman/cpp-lru-cache",
                        ":/licenses/lrucache.txt");
         }
+        
+        // About Chatterino
+        auto about_chatterino = layout.emplace<QGroupBox>("About Chatterino...");
```
Comment (pajlada) [inline src/widgets/settingspages/AboutPage.cpp:135]: rename this variable to be camelCase, so `aboutChatterino`

**Review by pajlada — CHANGES_REQUESTED** [review]
Would be nice to have the links defined at the top of this file as constexpr strings or something along those lines to make future changes easier

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,7 @@
 - Minor: Received Twitch messages now use the exact same timestamp (obtained from Twitch's server) for every Chatterino user instead of assuming message timestamp on client's side. (#3021)
 - Minor: Received IRC messages use `time` message tag for timestamp if it's available. (#3021)
 - Minor: Added informative messages for recent-messages API's errors. (#3029)
+- Minor: Added "About Chatterino" to About tab. (#2965, #3066)
```
Comment (pajlada) [inline CHANGELOG.md:10]: Could you add a small description of what's in the About Chatterino section? e.g. that it links to the wiki

**Review by pajlada — CHANGES_REQUESTED** [review]
Would still like the links in a constexpr or define at the top of the file, and the formatting seems to fail so please check your `clang-format`

**Comment by pajlada** [comment]
Another request: Please put the "About Chatterino" section above "Open source software used"

File: src/widgets/settingspages/AboutPage.cpp [hunk 4]
```diff
@@ -17,6 +17,11 @@
 
 #define PIXMAP_WIDTH 500
 
+#define chatterinoWiki "https://wiki.chatterino.com/"
+#define donateFourtf "https://streamelements.com/fourtf/tip/"
+#define chatterinoFeatures "https://chatterino.com/#features/"
+#define chatterinoDiscord "https://discord.gg/7Y5AYhAK4z/"
```
Comment (pajlada) [inline src/widgets/settingspages/AboutPage.cpp:23]:
```suggestion
#define LINK_CHATTERINO_WIKI "https://wiki.chatterino.com"
#define LINK_DONATE "https://streamelements.com/fourtf/tip"
#define LINK_CHATTERINO_FEATURES "https://chatterino.com/#features"
#define LINK_CHATTERINO_DISCORD "https://discord.gg/7Y5AYhAK4z"
```

File: src/widgets/settingspages/AboutPage.cpp [hunk 5]
```diff
@@ -97,6 +102,19 @@ AboutPage::AboutPage()
...
+        // About Chatterino
+        auto aboutChatterino = layout.emplace<QGroupBox>("About Chatterino...");
+        {
+            auto l = aboutChatterino.emplace<QVBoxLayout>();
+
+            // clang-format off
+            l.emplace<QLabel>("Chatterino Wiki can be found <a href=\""chatterinoWiki"\">here</a>")->setOpenExternalLinks(true);
+            l.emplace<QLabel>("Support <a href=\""donateFourtf"\">Chatterino</a>")->setOpenExternalLinks(true);
+            l.emplace<QLabel>("All about Chatterino's <a href=\""chatterinoFeatures"\">features</a>")->setOpenExternalLinks(true);
+            l.emplace<QLabel>("Join the official Chatterino <a href=\""chatterinoDiscord "\">Discord</a>")->setOpenExternalLinks(true);
```
Comment (pajlada) [inline src/widgets/settingspages/AboutPage.cpp]:
```suggestion
            l.emplace<QLabel>("Chatterino Wiki can be found <a href=\""LINK_CHATTERINO_WIKI"\">here</a>")->setOpenExternalLinks(true);
            l.emplace<QLabel>("Support <a href=\""LINK_DONATE"\">Chatterino</a>")->setOpenExternalLinks(true);
            l.emplace<QLabel>("All about Chatterino's <a href=\""LINK_CHATTERINO_FEATURES"\">features</a>")->setOpenExternalLinks(true);
            l.emplace<QLabel>("Join the official Chatterino <a href=\""LINK_CHATTERINO_DISCORD "\">Discord</a>")->setOpenExternalLinks(true);
```

**Review by pajlada — CHANGES_REQUESTED** [review]
Final changes (I think)

File: src/widgets/settingspages/AboutPage.cpp [hunk 6]
```diff
@@ -96,6 +102,20 @@ AboutPage::AboutPage()
...
+        // About Chatterino
+        auto aboutChatterino = layout.emplace<QGroupBox>("About Chatterino...");
+        {
+            auto l = aboutChatterino.emplace<QVBoxLayout>();
+
+            // clang-format off
+            l.emplace<QLabel>("Chatterino Wiki can be found <a href=\"" chatterinoWiki "\">here</a>")->setOpenExternalLinks(true);
+            l.emplace<QLabel>("Support <a href=\"" donateFourtf "\">Chatterino</a>")->setOpenExternalLinks(true);
+            l.emplace<QLabel>("All about Chatterino's <a href=\"" chatterinoFeatures "\">features</a>")->setOpenExternalLinks(true);
+            l.emplace<QLabel>("Join the official Chatterino <a href=\"" chatterinoDiscord "\">Discord</a>")->setOpenExternalLinks(true);
```
Comment (pajlada) [inline src/widgets/settingspages/AboutPage.cpp:114]:
```suggestion
            l.emplace<QLabel>("Chatterino Wiki can be found <a href=\"" LINK_CHATTERINO_WIKI "\">here</a>")->setOpenExternalLinks(true);
            l.emplace<QLabel>("Support <a href=\"" LINK_DONATE "\">Chatterino</a>")->setOpenExternalLinks(true);
            l.emplace<QLabel>("All about Chatterino's <a href=\"" LINK_CHATTERINO_FEATURES "\">features</a>")->setOpenExternalLinks(true);
            l.emplace<QLabel>("Join the official Chatterino <a href=\"" LINK_CHATTERINO_DISCORD "\">Discord</a>")->setOpenExternalLinks(true);
```

**Review by pajlada — APPROVED** [review]

File: CHANGELOG.md [hunk 7]
```diff
@@ -7,6 +7,7 @@
 - Minor: Received Twitch messages now use the exact same timestamp (obtained from Twitch's server) for every Chatterino user instead of assuming message timestamp on client's side. (#3021)
 - Minor: Received IRC messages use `time` message tag for timestamp if it's available. (#3021)
 - Minor: Added informative messages for recent-messages API's errors. (#3029)
+- Minor: Added "About Chatterino" to About tab where The Chatterino Wiki and other Stuff can be found. (#2965, #3068)
```
Comment (pajlada) [inline CHANGELOG.md:10]:
```suggestion
- Minor: Added section with helpful Chatterino-related links to the About page. (#3068)
```

**Review by zneix — APPROVED** [review]

**Pull request merged**